Dockerfile

```diff
@@ -115,17 +106,12 @@ ENV THRIFT_DEPS automake \
                 libboost-dev \
                 libboost-test-dev \
                 libevent-dev \
-                libssl1.0.0 \
                 libssl-dev \
                 libtool \
                 pkg-config \
-                python \
                 python3 \
-                python-dev \
                 python3-dev \
-                python-pip \
                 python3-pip \
-                python-setuptools \
                 python3-setuptools
```

Comment:
There is a package called [`python-is-python3`](https://packages.ubuntu.com/focal/python-is-python3) that might be a good idea to install. This will ensure that the commands using `/usr/bin/python` and Python scripts with `#!/usr/bin/env python` would continue to work.